lib/Dialect/FIRRTL/FIRRTLIntrinsics.cpp

```diff
@@ -25,6 +25,20 @@ ParseResult GenericIntrinsic::hasNInputs(unsigned n) {
   return success();
 }
 
+// Checks for a number of operands between n and m (allows for (n-m) optional
+// parameters)
+ParseResult GenericIntrinsic::hasNInputs(unsigned n, unsigned m) {
```

Comment:
my idea was to have something that said "between n and m parameters are allowed" rather than having "n params with c optional params". Both work, which one would you prefer?